client/src/index.js

```diff
@@ -1,21 +1,21 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
```

Comment:
We should get rid of this one, shoudn't we?

Reply:
:+1: